devtools-panels/devtools/panel/devtools-panel.js

```diff
@@ -0,0 +1,71 @@
+/**
+Handle errors from the injected script.
+Errors may come from evaluating the JavaScript itself
+or from the devtools framework.
+See https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/devtools.inspectedWindow/eval#Return_value
+*/
+function handleError(error) {
+  if (error.isError) {
+    console.log(`Devtools error: ${error.code}`);
+  } else {
+    console.log(`JavaScript error: ${error.value}`);
+  }
+}
+
+/**
+Handle the result of evaluating the script.
+If there was an error, call handleError.
+*/
+function handleResult(result) {
+  console.log('Function handleResult');
+  console.log(result);
+  if (result[0] !== undefined) {
+    console.log(`jQuery: ${result[0]}`);
```

Comment:
I think it's fragile to have this jQuery-specific handling in here. This function is called also to check the result of the other scripts: if one of them happens to not evaluate to undefined, then this will log misleading stuff.

It would probably be better to have a handler specifically for jQuery, then another generic one that only checks for errors.